Automattic/jetpack
# Admin Page: Remove some linting warnings

#### Changes proposed in this Pull Request:

* Renames `isNoticeDimissed` selector's argument to no collide with reducer name in jetpack-notices reducer file. Also removes some unused variables
* Removes quotes from property names which don't need them.
* Removes a few unnecessary spaces from several files


#### Testing instructions:

1. Check this branch
1. run `npm run lint`
1. Expect to see only `['catch']` related warnings
